lib/sneakers/version.rb

```diff
@@ -1,3 +1,3 @@
 module Sneakers
-  VERSION = "2.5.0"
+  VERSION = "2.5.1"
```

Comment:
Never ever bump version in a pull request. It's up to the maintainers to decide when and why it should be done.